### ISSUE TITLE
No luck with ios notifications...

### ISSUE THREAD

**Issue by jviure** [phonegap/phonegap-plugin-push#2660]
Hi! I've read many tutos but no one helps me, or I do not understand. With android works perfect this library but with ios... I cant receive nothing. 
I've the  next config. In js:

```
var push = PushNotification.init({
            android: {
            },
            browser: {
                pushServiceURL: 'http://push.api.phonegap.com/v1/push'
            },
            ios: {
                alert: 'true',
                badge: true,
                sound: 'false'
            },
            windows: {}
        });
```

In the firebase console I've downloaded the google.plist, this in the root of the project. In the config.xml  I've the <resource-file src="GoogleService-Info.plist" />

mmm I've uploaded to the firebase console -> ios section the APN auth key, I've write the Key ID and the Team ID, I suppose both correct.

I've tried to send notifications with the "tester" of the firebase console (in Android they works perfect!), but without luck.. they never arrive..

My php notification script..:

```
$fields = array
    (
        'to'  => "/topics/gridek",
        'data' => ["new_id" => $new_id,  'body'  => $title, 'title'     => 'News!',  'icon' => 'uei', 'priority' => 'high']
    );
     
    $headers = array
    (
        'Authorization: key=' . API_ACCESS_KEY,
        'Content-Type: application/json'
    );

    $ch = curl_init();
    curl_setopt( $ch,CURLOPT_URL, 'https://fcm.googleapis.com/fcm/send' );
...
```

What else can I see? is very frustrating!!

phonegap-plugin-push 2.2.3 "PushPlugin"
ios 10.3.3

Thanks!

**Comment by malwatte** [comment]
I believe iOS payload (FCM) should have a "notification",
```
{ "data": {
     "notification" : {
          "body" : "body",
          "title" : title",
          ...
        },
        ...
    }
```

**Comment by jviure** [comment]
Hi! thanks for your answer but still no luck.. I've tried this:

$fields = array
    (
        'to'  => "/topics/gridek",
        'data' => ["new_id" => $new_id,  'icon' => 'ei', 'priority' => 'high', "notification" => ['body'  => $title, 'title' => 'TEST']]
    );

But in IOS I does not receive nothing.. I've tried with Firebase console and I does not receive nothing too.. what can be?  
I HAVE A SUPOSSITION!  I don t see usually in samples code in this phorum what I do in js.. to subscribe to a topic, in Android works but seems that in IOS not. I've to save all the tokens of the devices that register (in an own bd..) and then  send the push to this registration_id's? I saw better the topic but.. seems in ios does not work. thanks
